### ISSUE TITLE
missing annotation on nginx-ingress-controller breaks install

### ISSUE THREAD

**Issue by eBeyond** [kubermatic/kubermatic#9244]
### What happened?
When I install kubermatic using the kubermatic-installer on hetzner the annotation "load-balancer.herzner.cloud/location=" on the nginx-ingress-controller is missing. The installer aborts. After setting the annotation and reexecuting the installer it finishes properly

### What should happen?
The installer should run only once. Means: The annotation is set properly by the installer or by the helm value of the ingress.

### How to reproduce?
Install with defaults on hetzner cloud.

### Environment
Fresh installation of KKP v2.19.2 on kube-one-cluster on hcloud

- KKP Version: v2.19.2
- Shared or separate master/seed clusters?: At this point the decision is not needed.
- Master/seed cluster provider: kube-one-cluster running on hcloud

**Comment by xrstf** [comment]
Shouldn't you be able to set the annotations via https://github.com/kubermatic/kubermatic/blob/master/charts/nginx-ingress-controller/values.yaml#L79-L81 ?

**Comment by eBeyond** [comment]
Yes, of course. I've already done this using the values.yaml file. 
But for me, it looks more or less like a bug. You're running an installer, which sets everything, except this one annotation. The informaiton, the dc, is present, while installing and I think this one annotation can also be provided using already existing mechanisms.

**Comment by xrstf** [comment]
> You're running an installer, which sets everything, except this one annotation. 

Just to be clear, you don't expect the installer to magically auto-detect the necessary annotations, right? :)

> But for me, it looks more or less like a bug.

Absolutely, if the configured annotations are borked, we should fix that.

**Comment by eBeyond** [comment]
Oh, some "mysitc processing" would be great.  :D

But I think if it's possible to set the proper annotaion on nodeport-proxy, with the used datacenter, it should also be possible to do that on the nginx-ingress-controller (if it's installed by the installer).

That sounds not so magic. ;)

**Comment by xrstf** [comment]
Certainly, and if your Helm config files contain the annotation and they are lost, that's a bug. Can you show us your config files?
